test/spec/node-index.js

```diff
@@ -5,3 +5,4 @@ import './test-terminate.js'
 import './test-web-stream.js'
 import './test-binary-data.js'
 import './test-end-to-end.js'
+import './test-stall-detection.js'
```

Comment:
Is this test specific to Node.js? If not, it should be added to `test-common.js`, so it's also run in automated browsers.